### PR TITLE
Miscellaneous code inspection fixes

### PR DESCRIPTION
**Pull request by mnonnenmacher**
Please see the commit messages for details.

### CODE REVIEW

File: clients/github-graphql/src/test/kotlin/GitHubServiceTest.kt [hunk 1]
```diff
@@ -48,6 +48,9 @@ import java.net.ServerSocket
 import java.net.URI
 import java.util.regex.Pattern
 
+import kotlinx.coroutines.Dispatchers
```
Comment (sschuberth) [inline clients/github-graphql/src/test/kotlin/GitHubServiceTest.kt:51]: Commit message: Typo in "exectues"

File: analyzer/src/main/kotlin/managers/Unmanaged.kt [hunk 2]
```diff
@@ -69,17 +69,15 @@ class Unmanaged(
             vcsInfo == VcsInfo.EMPTY -> {
                 // This seems to be an analysis of a local directory that is not under version control, i.e. that is not
                 // a VCS working tree. In this case we have no change to get a version.
-                val projectDir = definitionFile
-
                 log.warn {
-                    "Analysis of local directory '$projectDir' which is not under version control will produce " +
+                    "Analysis of local directory '$definitionFile' which is not under version control will produce " +
```
Comment (sschuberth) [inline analyzer/src/main/kotlin/managers/Unmanaged.kt:73]: FYI: The variable was actually introduced intentionally to make it explicit that in this case not a *file* but a *dir* is being handled. But maybe the mention of "directory" in the warn log is enough.

File: clients/nexus-iq/src/main/kotlin/NexusIqService.kt [hunk 3]
```diff
@@ -62,7 +62,7 @@ interface NexusIqService {
         /**
          * The JSON (de-)serialization object used by this service.
          */
-        val JSON = Json { ignoreUnknownKeys = true }
+        private val JSON = Json { ignoreUnknownKeys = true }
```
Comment (sschuberth) [inline clients/nexus-iq/src/main/kotlin/NexusIqService.kt:65]: By convention these should stay public / be part of the public API, so consumers of the clients can use exactly the same serializer as the client code itself.

File: clients/oss-index/src/main/kotlin/OssIndexService.kt [hunk 4]
```diff
@@ -45,7 +45,7 @@ interface OssIndexService {
         /**
          * The JSON (de-)serialization object used by this service.
          */
-        val JSON = Json.Default
+        private val JSON = Json.Default
```
Comment (sschuberth) [inline clients/oss-index/src/main/kotlin/OssIndexService.kt:48]: See above.

File: clients/vulnerable-code/src/main/kotlin/VulnerableCodeService.kt [hunk 5]
```diff
@@ -41,7 +41,7 @@ interface VulnerableCodeService {
         /**
          * The JSON (de-)serialization object used by this service.
          */
-        val JSON = Json { ignoreUnknownKeys = true }
+        private val JSON = Json { ignoreUnknownKeys = true }
```
Comment (sschuberth) [inline clients/vulnerable-code/src/main/kotlin/VulnerableCodeService.kt:44]: See above.

**Review by sschuberth — CHANGES_REQUESTED** [review]

File: analyzer/src/main/kotlin/managers/Unmanaged.kt (hunk 2 above)
Comment (mnonnenmacher) [reply analyzer/src/main/kotlin/managers/Unmanaged.kt:73]: I would think so, as it's also explained in the code comment.

**Review by sschuberth — APPROVED** [review]

**Pull request merged**